tslint.json

```diff
@@ -0,0 +1,3 @@
+{
+  "extends": "gts/tslint.json", "linterOptions": {"exclude": []}
```

Comment:
No need to have an empty `linterOptions` :)

Reply:
sure, removed the `linterOptions` part